src/main/java/org/broadinstitute/hellbender/utils/genotyper/AlleleLikelihoods.java

```diff
@@ -1092,7 +1092,7 @@ private BestAllele(final int sampleIndex, final int evidenceIndex, final int bes
         }
 
         public boolean isInformative() {
-            return confidence > LOG_10_INFORMATIVE_THRESHOLD;
+            return confidence > getInformativeThreshold();
```

Comment:
Is this code shared with `HaplotypeCaller` (which is still on log10) @davidbenjamin / @jamesemery ?

Reply:
Yes, but `getInformativeThreshold()` knows whether the likelihoods are base-10 or natural log.